Resources/Prototypes/_White/Recipes/Lathes/devices.yml

```
@@ -0,0 +1,19 @@
+    Silver: 100
+    Gold: 100
+
+- type: latheRecipe
+  id: ClothingEyesThermalVisionGoggles
+  result: ClothingEyesThermalVisionGoggles
+  completetime: 2
+  materials:
+    Steel: 200
+    Glass: 100
+    Silver: 100
+    Gold: 100
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)